### PR TITLE
[ML] Fix VS linking problem

### PR DESCRIPTION
**Pull request by valeriy42**
Since VS was complaining about multiple exported instances of `CRapidJsonLineWriter`, I changed the code in order to use `CRapidJsonConcurrentLineWriter` as an instantiation of `CRapidJsonLineWriter`.

### CODE REVIEW

**Comment by droberts195** [comment]
retest

File: lib/api/unittest/CBoostedTreeInferenceModelBuilderTest.cc [hunk 1]
```diff
@@ -256,7 +256,7 @@ void CBoostedTreeInferenceModelBuilderTest::testIntegrationClassification() {
 }
 
 void CBoostedTreeInferenceModelBuilderTest::testJsonSchema() {
-    std::size_t numberExamples = 1000;
+    std::size_t numberExamples = 200;
```
Comment (przemekwitek) [inline lib/api/unittest/CBoostedTreeInferenceModelBuilderTest.cc]: Why is this change in this PR? Is it somehow related?

Comment (valeriy42) [reply lib/api/unittest/CBoostedTreeInferenceModelBuilderTest.cc]: No you a right, it's unrelated, I will move it to another PR.

File: include/core/CRapidJsonLineWriter.h [hunk 2]
```diff
@@ -29,7 +29,7 @@ template<typename OUTPUT_STREAM,
          typename TARGET_ENCODING = rapidjson::UTF8<>,
          typename STACK_ALLOCATOR = rapidjson::CrtAllocator,
          unsigned WRITE_FLAGS = rapidjson::kWriteDefaultFlags>
-class CORE_EXPORT CRapidJsonLineWriter
```
Comment (droberts195) [inline include/core/CRapidJsonLineWriter.h]: Please can you also remove the header from line 12.

Comment (valeriy42) [reply include/core/CRapidJsonLineWriter.h]: Done

Comment (valeriy42) [reply include/core/CRapidJsonLineWriter.h]: Done

**Comment by valeriy42** [comment]
retest

**Comment by valeriy42** [comment]
retest

**Pull request merged**